beeminder_test.go

```
@@ -2278,6 +2278,107 @@ func TestCallUncleWithMockServer(t *testing.T) {
+	t.Run("successful ratchet", func(t *testing.T) {
+		mockServer := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+			if r.Method != http.MethodPost {
+				t.Errorf("Expected POST request, got %s", r.Method)
+			}
+
+			expectedPath := "/api/v1/users/testuser/goals/testgoal/ratchet.json"
+			if r.URL.Path != expectedPath {
+				t.Errorf("Expected path %s, got %s", expectedPath, r.URL.Path)
+			}
+
+			r.ParseForm()
```

golangci-lint (2.12.2)

[error] Error return value of `r.ParseForm` is not checked

(errcheck)